src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -573,19 +573,19 @@ def handle_migration_parameters(command_group, server_name_arg_type, migration_i
                     c.argument('properties', options_list=['--properties', '-b'],
                                help='Request properties. Use @{file} to load from a file. For quoting issues in different terminals, '
                                'see https://github.com/Azure/azure-cli/blob/dev/doc/use_cli_effectively.md#quoting-issues')
-                    c.argument('migration_id', arg_type=migration_id_arg_type, options_list=['--migration-id'],
-                               help='Name or ID of the migration.')
+                    c.argument('migration_name', arg_type=migration_id_arg_type, options_list=['--migration-name'],
+                               help='Name of the migration.')
```

Comment:
Can you query `name` out of the provided `id`? It's better to not cause breaking change for CX. Both `id` and `name` are supported before.

Reply:
it will have to be `migration_name` because we are using `migration_id` in the server-side code for this migration workflow. The changes have already been shipped for this. Also, the APIs and future functionality are written with respect to `migration_name` as provided by the customer. Although, the `migration_name` can accept any strings(read guid's).

Reply:
It's a breaking change for cx's existing scripts.

I can help merge if you insist